puppet/vagrant/modules/socorro/files/build_python.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This script downloads Python source tarball, builds it and does an altinstall.
+# An altinstall installs Python, but doesn't change any of the symlinks in
+# /usr/bin, so it doesn't affect anything else on the system that is expecting
+# Python 2.7.5.
+
+PYVERSION=2.7.11
+
+cd /tmp
```

Comment:
for this and the next one, you could also consider `set -e`, which will exit on a failed cd _most_ of the time